### ISSUE TITLE
Откатить питон на 3.8 и добавить tf 2.x

### ISSUE THREAD

**Issue by deniskolsanov** [MailRuChamps/raic-2020#21]
tensorflow 2.3 cpu only Linux for python 3.8

pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow_cpu-2.3.0-cp38-cp38-manylinux2010_x86_64.whl

**Comment by oPOCCOMAXAo** [comment]
нет, не нужны либы с нейронками, т.к. в других языках этой возможности нет.
@deniskolsanov если вам нужны нейросети, то пишите их с нуля

**Comment by deniskolsanov** [comment]
@oPOCCOMAXAo ну раз вам не нужны либы с нейронккми, то и не пользуйтесь либами с нейронками. В других языках есть возможность попросить добавить либу, но никто не просит, потому что им по всей видимости они не нужны

**Comment by SSS135** [comment]
@oPOCCOMAXAo Не надо бить лежачего, нейронки и так призовые места не занимали никогда. Хорошо бы ещё PyTorch добавить.
pip install torch==1.7.0+cpu torchvision==0.8.1+cpu torchaudio==0.7.0 -f https://download.pytorch.org/whl/torch_stable.html

**Comment by Hammer2900** [comment]
Автор че ты такой жадный, да хочет нейронку, пускай используют, это же весело. Тебя это что сильно напряжет.

**Comment by AKonia** [comment]
про откат питона идея хорошая, т.к. 3.9 не поддерживается в Вин7
